### DISCUSSION TITLE
JS updates when packages are part of CSS/JS building

### DISCUSSION THREAD

**Discussion by back-2-95** [renovatebot/renovate#11162] [Suggest an Idea]
We have Drupal themes which have source JS and SASS CSS.

Then we might have Gulp (older projects) or Webpack (newer projects) to build CSS/JS (dist).
So these JS packages are not actual code running on the Drupal website (vs e.g. React components etc).

I was thinking what maybe updates in this scenario could be ignored. As otherwise we'll end up with lots of JS updates to our css/js building toolchain in multiple repos where many updates will require manual changes too.

What's your take on this on same or similar scenarios?

**Comment by back-2-95** [comment]
This is quite normal example in our projects:

![CleanShot 2021-08-08 at 11 44 48](https://user-images.githubusercontent.com/1140272/128626415-e4445e7e-6afd-48a2-8307-63e483853078.png)

**Reply by back-2-95** [reply]
All the JS packages could be ignored as they are just part of the toolchain to build dist CSS/JS.

**Comment by rarkins** [comment]
First question is whether or not they should be updated.

If no, then the next question is whether Renovate could/should somehow detect that. 

If yes, then perhaps you want to group them if you want to reduce the number of PRs, but that is not something we have an opinion on.

**Reply by back-2-95** [reply]
This was not about Renovate detecting it. This was more just a question how people approach JS packages in this kind of scenario. I can ignore path if I think this is the way to go.

Just looking for what others have thought and done.
